webapp/javascript/hooks/timeline.hook.ts

```diff
@@ -6,12 +6,15 @@ import {
   selectTimelineSidesData,
 } from '@webapp/redux/reducers/continuous';
 import Color from 'color';
+import useCancelRequestOnUnmount from '@webapp/hooks/cancelRequestOnUnmount.hook';
 
 // Purple
 export const leftColor = Color('rgb(208, 102, 212)');
 // Blue
 export const rightColor = Color('rgb(19, 152, 246)');
 
+let timelineData: ShamefulAny;
```

Comment:
Ugh, isn't there a way to type this? Maybe with `ReturnType<dispatch>`?

Reply:
there is no this code in last version of this file, but there are places when I left this problem. let me check